packages/runtime/src/next/components/tests/controls/style-control/__snapshots__/server.test.tsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders a style: component styles 1`] = `
+[
+  ".css-1cny9vj{max-width:100%;}@media only screen and (min-width: 769px){.css-1cny9vj{width:80%;margin-top:4px;margin-right:auto;margin-bottom:0px;margin-left:auto;}}@media only screen and (min-width: 576px) and (max-width: 768px){.css-1cny9vj{width:80%;margin-top:4px;margin-right:auto;margin-bottom:0px;margin-left:auto;}}@media only screen and (max-width: 575px){.css-1cny9vj{width:80%;margin-top:4px;margin-right:auto;margin-bottom:0px;margin-left:auto;}}",
+]
+`;
+
+exports[`renders a style: resolvedValue 1`] = `"css-1cny9vj u-R0-propKey"`;
```

Comment:
New snapshot documenting that style generation works as expected. 

Here's what a failing test looks like:

<img width="1654" alt="failing snapshot" src="https://github.com/user-attachments/assets/8c6052f2-23a7-4d13-8812-97cfcc5b7263" />